examples/2d/tilemap_chunk.rs

```diff
@@ -0,0 +1,88 @@
+//! Shows a tilemap chunk rendered with a single draw call.
+
+use bevy::{
+    dev_tools::fps_overlay::FpsOverlayPlugin,
+    prelude::*,
+    sprite::{TilemapChunk, TilemapChunkIndices},
+};
+use rand::{Rng, SeedableRng};
+use rand_chacha::ChaCha8Rng;
+
+fn main() {
+    App::new()
+        .add_plugins((
+            DefaultPlugins.set(ImagePlugin::default_nearest()),
+            FpsOverlayPlugin::default(),
```

Comment:
This should be removed from the example. It's nice for development but not for users looking at the example.

I also want to avoid users complaining about poor performance when the issue would likely just be that vsync is enabled by default and not actually low fps.